### ISSUE TITLE
deprecate setup in webwork

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#710]
With the forthcoming scheme for webwork elements, there is no need for var/static as children of setup. This means setup will have only one child: pg-code. So we can just eliminate setup.

Fixing deprecations would mean deleting all setup/var and then moving setup/pg-code to just pg-code, deleting setup. Children of a webwork would be:
title?,pg-macros?,pg-code?,statement,hint*,solution*

This is just a reminder to do this at some point.

**Comment by rbeezer** [comment]
Will the change be backward-compatible?  In other words can the XSL look for `pg-code` both ways?  I can add a warning for the old way.

Note to myself:  `fix-deprecations.xsl` will go something like
```xsl
<xsl:template match="setup">
  <xsl:apply-templates select="pg-code" />
</xsl:template>
```

**Comment by Alex-Jordan** [comment]
I think it's easy enough to make the pg-code part backward compatible. But I don't want to make backward compatible the part about using author-declared values for static variables.

**Comment by rbeezer** [comment]
Right - the old setup/var can just vanish, as I understand it.

You'll just have to test for "setup" being present and get "pg-code" from there 
in that case, otherwise get it from new arrangement.

**Comment by Alex-Jordan** [comment]
See #1261.

**Comment by rbeezer** [comment]
Addressed on #1261.
